layouts/fastly/helix.vcl

```diff
@@ -1999,9 +2004,15 @@ sub hlx_bereq {
     # set Adobe Runtime backend authentication
     set bereq.http.Authorization = table.lookup(secrets, "OPENWHISK_AUTH");
     # pass Github Token via X-Github-Token header
-    set bereq.http.X-Github-Token = table.lookup(secrets, "GITHUB_TOKEN");
+    if (!req.http.X-Github-Token) {
+      # use global token if header not set
+      set bereq.http.X-Github-Token = table.lookup(secrets, "GITHUB_TOKEN");
```

Comment:
I think having a github token per repository stored in the table would be better. something like:

```
set bereq.http.X-Github-Token = table.lookup(githubtoken, repo)
```

Reply:
That leaves us with the problem of keeping the table updated, which would not scale well and cause issues with key rotation, taking sites offline, etc. It also shifts the frontier of customization from the outer CDN to the inner CDN.

Reply:
currently there is 1 customer.